cards/src/main/resources/cards/hearthstone/journey_to_ungoro/druid/spell_living_mana.json

```diff
@@ -7,43 +7,7 @@
   "description": "Transform your Mana Crystals into 2/2 minions. Recover the mana when they die.",
   "targetSelection": "NONE",
   "spell": {
-    "class": "MetaSpell",
-    "value": {
-      "class": "AlgebraicValueProvider",
-      "operation": "MINIMUM",
-      "value1": {
-        "class": "PlayerAttributeValueProvider",
-        "playerAttribute": "MAX_MANA",
-        "targetPlayer": "SELF"
-      },
-      "value2": {
-        "class": "AlgebraicValueProvider",
-        "operation": "SUBTRACT",
-        "value1": 7,
-        "value2": {
-          "class": "EntityCounter",
-          "target": "FRIENDLY_MINIONS"
-        }
-      }
-    },
-    "spells": [
-      {
-        "class": "SummonSpell",
-        "value": {
-          "class": "GameValueProvider",
-          "gameValue": "SPELL_VALUE"
-        },
-        "card": "token_mana_treant"
-      },
-      {
-        "class": "ModifyMaxManaSpell",
-        "value": {
-          "class": "GameValueProvider",
-          "gameValue": "SPELL_VALUE",
-          "multiplier": -1
-        }
-      }
-    ]
+    "class": "ManaCrystalToMinionSpell"
```

Comment:
this should probably be a `custom.ManaCrystalToMinionSpell`. Also, specify the minion it should be here using `"card": "..."` and use it in the body of the `onCast` in `ManaCrystalToMinionSpell` by retrieving the card using `SpellUtils.getCard(desc...)`